tools/src/main/scala/scala/scalanative/build/ScalaNative.scala

```diff
@@ -184,4 +184,7 @@ private[scalanative] object ScalaNative {
       Global.Top(encoded)
     }
 
+  def identgen(config: Config): Seq[java.nio.file.Path] =
```

Comment:
The generated info is very similar to things we might get from [buildInfo](https://github.com/sbt/sbt-buildinfo) on the JVM. How about if we would keep this naming convention, and possibly extend it in the future if needed
`codegen` is quite a widely used name, but `identgen` is a bit exotic for me. Maybe we can combine both of these though to make it more straight forward?
```suggestion
  def genBuildInfo(config: Config): Seq[java.nio.file.Path] =
```